tqp/raven.py

```diff
@@ -9,6 +9,6 @@ def install(client):
     def _hook_handle_error(self, exception, msg, payload):
         base_handle_error(self, exception, msg, payload)
 
-        client.captureException(extra={"msg": msg})
```

Comment:
@itajaja the problem is that this is actually some opaque object from moto, so nothing useful will go to sentry

Reply:
oh ok. as long as payload contains the queue name?

Reply:
well, it has `topic = body['TopicArn'].split(':')[-1]`